homeassistant/components/squeezebox/media_player.py

```diff
@@ -587,4 +590,23 @@ async def async_browse_media(self, media_content_type=None, media_content_id=Non
             "search_id": media_content_id,
         }
 
-        return await build_item_response(self._player, payload)
+        return await build_item_response(self, payload)
+
+    async def async_get_browse_image(self, browse_image):
+        """Get album art from Squeezebox server."""
+        internal_url = f"{self._internal_artwork_url}{browse_image}"
+        result = await self._async_fetch_image(internal_url)
```

Comment:
Now the browse images also end up in the cache, we shouldn't do that as they are less likely to be reused.